bokehjs/src/coffee/common/base.coffee

```diff
@@ -130,20 +131,8 @@ locations =
   HoverTool:                require '../tool/inspectors/hover_tool'
   InspectTool:              require '../tool/inspectors/inspect_tool'
 
-  StringFormatter:          require('../widget/cell_formatters').String
-  NumberFormatter:          require('../widget/cell_formatters').Number
-  BooleanFormatter:         require('../widget/cell_formatters').Boolean
-  DateFormatter:            require('../widget/cell_formatters').Date
-
-  StringEditor:             require('../widget/cell_editors').String
-  TextEditor:               require('../widget/cell_editors').Text
-  SelectEditor:             require('../widget/cell_editors').Select
-  PercentEditor:            require('../widget/cell_editors').Percent
-  CheckboxEditor:           require('../widget/cell_editors').Checkbox
-  IntEditor:                require('../widget/cell_editors').Int
-  NumberEditor:             require('../widget/cell_editors').Number
-  TimeEditor:               require('../widget/cell_editors').Time
-  DateEditor:               require('../widget/cell_editors').Date
+  editors:                  [require('../widget/cell_editors'), "Editor"]
```

Comment:
To make sure I understand what's going on here.

* This loads the exports for `widget/cell_editors`
* Iterates over all of the exposed modules and assigns ``{{ some name }}` to `{{ some name }}Editor` for external access

FWIW, that's not immediately obvious.  Took a little back and forth and code reading to understand what was at play.